From: Michael Biebl <biebl@debian.org>
Date: Tue, 19 Nov 2019 09:10:23 +0100
Subject: udev: drop SystemCallArchitectures=native from systemd-udevd.service

We can't really control what helper programs are run from other udev
rules. E.g. running i386 binaries under amd64 is a valid use case and
should not trigger a SIGSYS failure.

Closes: #869719
---
 units/systemd-udevd.service.in | 1 -
 1 file changed, 1 deletion(-)

--- a/units/systemd-udevd.service.in
+++ b/units/systemd-udevd.service.in
@@ -37,7 +37,6 @@
 SystemCallFilter=@system-service @module @raw-io bpf
 SystemCallFilter=~@clock
 SystemCallErrorNumber=EPERM
-SystemCallArchitectures=native
 LockPersonality=yes
 IPAddressDeny=any
 {{SERVICE_WATCHDOG}}
